Exercícios/Para casa/entregas/mariana_inoue/BankAccount.js

```diff
@@ -0,0 +1,67 @@
+const { Bank } = require("./Bank");
+const { Client } = require("./Client");
+
+class BankAccount {
+  accountNumber;
+  agencyNumber;
+  #balance = 0;
+  client;
+  bank;
+
+  constructor(client, bank, accountNumber, agencyNumber) {
```

Comment:
Olá, Mari. Vendo o seu código eu percebo que a validação  de this.client e this.bank como instâncias de Client e Bank era mais simples do que eu pensava. Tive dificuldade em realizar essa validação no meu.